superset/models.py

```diff
@@ -871,6 +872,7 @@ class SqlaTable(Model, Queryable, AuditMixinNullable, ImportMixin):
     default_endpoint = Column(Text)
     database_id = Column(Integer, ForeignKey('dbs.id'), nullable=False)
     is_featured = Column(Boolean, default=False)
+    filter_select_enabled = Column(Boolean, default=False)
```

Comment:
I think the original incentive was that filter method only retrieves samples of the values, not all of them, for some columns there could be way too many values to be displayed, and user prefer to directly add values in input, separated by comma, instead of wrapping each value in single quote and add it as an option. (The reason for wrapping values in quote is to accomodate values with comma in them)